templates/dashboard.html

```diff
@@ -14,6 +14,16 @@ <h5 class="card-title">{{ config["ASSET_NAME"] }} balance</h5>
             </div>
         </div>
 
+        <div class="card">
```

Comment:
lets move this into the asset balance card above

Reply:
moved.